flocker/control/test/test_protocol.py

```diff
@@ -654,3 +654,29 @@ def test_logging(self, logger):
                 "agent": connection_protocol,
             }
         )
+
+    @validate_logging(None)
+    def test_error_sending(self, logger):
+        """
+        An error sending to one agent does not prevent others from being
+        notified.
+        """
+        control_amp_service = build_control_amp_service(self)
+        self.patch(control_amp_service, 'logger', logger)
+
+        connection_protocol = ControlAMP(control_amp_service)
+        connection_protocol.callRemote = lambda *args, **kwargs: succeed({})
```

Comment:
`connected_protocol` is probably the name that parallels `disconnected_protocol` best.